Readme.md

```diff
@@ -1 +1,227 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+  - Set  up project with create-react-app
+  - Create home and details routes 
+  - Fetch weather data from openWeatherMap end point
+  - Display weather data in home and weather details in details component 
+  - Implement search functionality
+ 
+    
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Weather Application](https://weather-application-dt9z.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ npm install 
+```
+
+```sh
+ npm start
+```
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/biruk-bereka/weather-application.git
+```
+-
+
+### Install
+
+Install dependencies: 
+
+```sh
+npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+npm test
+```
+
+To run the stylehint linter
+```sh
+ npx stylelint "**/*.{css,scss}"
+```
+
+To run the ESLint linter
+```sh
+  npx eslint "**/*.{js,jsx}"
+```
+### Deployment
+
+You can deploy this project using:
+```sh
+  npm run build 
+```
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Biruk Bereka**
+
+- GitHub: [@biruk-bereka](https://github.com/biruk-bereka)
+- LinkedIn:[@Biruk Bereka](https://www.linkedin.com/in/biruk-bereka1212/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Making improvements ]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star!! 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
```

Comment:
- [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your `Acknowledgement` section of your readme file